actors/v2/verifiedRegistry/params.go

```diff
@@ -18,19 +20,76 @@ import (
 	legacyv6 "github.com/filecoin-project/specs-actors/v6/actors/builtin/verifreg"
 	legacyv7 "github.com/filecoin-project/specs-actors/v7/actors/builtin/verifreg"
 	cbg "github.com/whyrusleeping/cbor-gen"
+	"github.com/zondax/fil-parser/actors"
 	"github.com/zondax/fil-parser/actors/v2/verifiedRegistry/types"
+	"github.com/zondax/fil-parser/parser"
 	"github.com/zondax/fil-parser/tools"
 )
 
+func v1Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	v := &VerifiedRegistry{}
+	return map[abi.MethodNum]nonLegacyBuiltin.MethodMeta{
+		1: {
+			Name:   parser.MethodConstructor,
+			Method: actors.ParseConstructor,
+		},
+		2: {
+			Name:   parser.MethodAddVerifier,
+			Method: v.AddVerifier,
+		},
+		3: {
+			Name:   parser.MethodRemoveVerifier,
+			Method: v.RemoveVerifier,
+		},
+		4: {
+			Name:   parser.MethodAddVerifiedClient,
+			Method: v.AddVerifiedClientExported,
+		},
+		5: {
+			Name:   parser.MethodUseBytes,
+			Method: v.UseBytes,
+		},
+		6: {
+			Name:   parser.MethodRestoreBytes,
+			Method: v.RestoreBytes,
+		},
+	}
+}
+
+func v2Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	return v1Methods()
+}
+func v3Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	return v1Methods()
+}
+func v4Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	return v1Methods()
+}
+func v5Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	return v1Methods()
+}
+func v6Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	return v1Methods()
+}
+func v7Methods() map[abi.MethodNum]nonLegacyBuiltin.MethodMeta {
+	v := &VerifiedRegistry{}
+	methods := v6Methods()
+	methods[7] = nonLegacyBuiltin.MethodMeta{
+		Name:   parser.MethodRemoveVerifiedClientDataCap,
+		Method: v.RemoveVerifiedClientDataCap,
+	}
+	return methods
```

Comment:
I guess we do this to allow possible extensibility, right? as they end up being practically the same as before this PR. Am I right?